recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/0001-Revert-clk-renesas-r8a7795-Remove-clock-that-supplie.patch

```diff
@@ -0,0 +1,27 @@
+From d9d3ba03c09648ba5416288b6cb7aab616c1ffaa Mon Sep 17 00:00:00 2001
+From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
+Date: Tue, 14 Aug 2018 21:41:24 +0300
+Subject: [PATCH 1/4] Revert "clk: renesas: r8a7795: Remove clock that supplies
+ all thermal sensors"
```

Comment:
Shouldn't we just revert those patches from LK, rather than introducing this stuff into yocto?
As I remember we considered it a bad practice to have patches for LK, XEN, etc in yocto layers.

Reply:
I don't think we should do this right now. If we revert patches from LK we probably get prod-devel broken